elliott/elliottlib/runtime.py

```diff
@@ -94,6 +94,17 @@ def __init__(self, **kwargs):
     def get_major_minor(self):
         return self.group_config.vars.MAJOR, self.group_config.vars.MINOR
 
+    def get_major_minor_patch(self):
+        pattern = r"\d+.\d+.\d+"
+        match = re.search(pattern, self.assembly)
+        if match:
+            self._logger.debug(f"Found major.minor.patch for assembly {self.assembly}")
```

Comment:
how about also logging the found value here?

Reply:
ack added in https://github.com/openshift-eng/art-tools/pull/1573/commits/588bcdd0118448bef4189750bf0d00936f20122d